Files/BaseLayout.cs

```diff
@@ -420,9 +420,17 @@ protected override async void OnNavigatedTo(NavigationEventArgs eventArgs)
 
             ItemManipulationModel.FocusFileList(); // Set focus on layout specific file list control
 
+            SetSelectedItemsOnNavigation();
+
+            ItemContextMenuFlyout.Opening += ItemContextFlyout_Opening;
+            BaseContextMenuFlyout.Opening += BaseContextFlyout_Opening;
```

Comment:
I know these aren't new, but can you confirm if we unsubscribe from these events?

Reply:
Yep, in OnNavitagingFrom